s3client.go

```diff
@@ -317,13 +318,13 @@ func (c *s3Client) upload(r io.Reader) error {
 	key := c.config.S3.KeyConstructor(c.now, uid)
 	c.debugf("uploading to s3://%s/%s", c.config.S3.Bucket, key)
 
-	input := &s3manager.UploadInput{
+	input := &s3.PutObjectInput{
 		Body:   r,
 		Bucket: aws.String(c.config.S3.Bucket),
-		ACL:    aws.String("public-read"),
+		ACL:    types.ObjectCannedACLPublicRead,
```

Comment:
I think we should remove setting ACLs altogether - we stopped using ACLs, this functionality is deprecated by AWS.